packages/libuild/libuild-core/.mocharc.js

```diff
@@ -0,0 +1,7 @@
+const path = require('path');
```

Comment:
We should migrate from mocha to vitest later.

Reply:
It won't be included in this PR.